pandas/io/formats/style.py

```diff
@@ -1512,26 +1585,24 @@ def _background_gradient(
         text_color_threshold: float = 0.408,
         vmin: Optional[float] = None,
         vmax: Optional[float] = None,
+        gmap: Optional[Union[Sequence, np.ndarray, FrameOrSeries]] = None,
```

Comment:
this need not be a staticmethod which is adding all kind of complexity here, make this a method as you now need to pass axis

Reply:
this was a static method before I made modifications. It does not need to be a static method so have taken it module level.